pkg/asset/installconfig/azure/validation_test.go

```diff
@@ -349,7 +351,10 @@ func Test_validateResourceGroup(t *testing.T) {
 		err:       `^\Qplatform.azure.resourceGroupName: Invalid value: "valid-resource-group-conf-tags": resource group has conflicting tags kubernetes.io_cluster.test-cluster-12345\E$`,
 	}, {
 		groupName: "valid-resource-group-with-resources",
-		err:       `^\Qplatform.azure.resourceGroupName: Invalid value: "valid-resource-group-with-resources": resource group must be empty but it has 3 resources like id1, id2 ...\E$`,
+		// ARO provisions Azure resources before resolving the asset graph,
+		// so there will always be resources in its resource group.
+		wantSkip: p.IsARO(),
```

Comment:
The `p` variable is too far away to use the name `p` for the variable. So either rename the variable so that it is clear what the variable is, or don't use a variable.
```suggestion
		wantSkip: (&azure.Platform{}).IsARO(),
```